src/js/twitter.js

```diff
@@ -1,3 +1,5 @@
+import {getNodesDeep} from 'utils.js'
```

Comment:
Makes it `import { getNodesDeep } from 'utils'` please.

Reply:
Had a look at the other files while trying to fix this issue (I didn't see this suggested change) and they've used `./utils` so I've used that instead - seems to work